sklearn/metrics/_scorer.py

```diff
@@ -214,14 +203,13 @@ def _get_pos_label(self):
         return None
 
     def __repr__(self):
-        kwargs_string = "".join(
-            [", %s=%s" % (str(k), str(v)) for k, v in self._kwargs.items()]
-        )
-        return "make_scorer(%s%s%s%s)" % (
-            self._score_func.__name__,
-            "" if self._sign > 0 else ", greater_is_better=False",
-            self._factory_args(),
-            kwargs_string,
+        sign_string = "" if self._sign > 0 else ", greater_is_better=False"
+        response_method_string = f", response_method={self._response_method!r}"
```

Comment:
Compare to the past `repr`, now I am always showing the `response_method` used. Before, we would not show the `"predict"` case. I think this is more explicit.